subsys/bluetooth/host/att.c

```diff
@@ -103,6 +103,10 @@ struct bt_att {
 #endif
 	/* Contains bt_att_chan instance(s) */
 	sys_slist_t		chans;
+	struct k_work_delayable connection_work;
+	struct att_connection_work_context {
```

Comment:
Is there a better way to pass information to the handler?

Reply:
If we just want to pass `ecred_chans_to_connect`, we don't need to put that in a struct, but otherwise this is the easiest way. 

One alternative is of course to put both `struct k_work_delayable connection_work;` and `int ecred_chans_to_connect;` into the `connection_work_context` struct, such that instead of getting a `bt_att` from `CONTAINER_OF`, we just get a `struct att_connection_work_context`. It will basically end up being the same though.